packages/patternfly-4/react-core/src/components/Checkbox/Checkbox.d.ts

```diff
@@ -1,11 +1,22 @@
-import { HTMLProps, FormEvent } from 'react';
+import PropTypes from 'prop-types';
+import { HTMLProps, FormEvent, ReactNode } from 'react';
 import { Omit } from '../../typeUtils';
+import { OneOf } from '../../typeUtils';
+
+export const TextVariants: {
+  string: string;
+  node: ReactNode;
+  function: Function;
+};
 
 export interface CheckboxProps
   extends Omit<HTMLProps<HTMLInputElement>, 'type' | 'onChange' | 'disabled'> {
   isDisabled?: boolean;
   isValid?: boolean;
   onChange?(checked: boolean, event: FormEvent<HTMLInputElement>): void;
+  id?: string;
+  'aria-label'?: string;
+  label?: OneOf<typeof TextVariants, keyof typeof TextVariants>;
```

Comment:
This is more easily written as `string | ReactNode | Function`. That being said I am curious why function is needed at all. Is there a usecase for it. I understand using render functions, but for a simple label is seems to add complications where they are not needed.